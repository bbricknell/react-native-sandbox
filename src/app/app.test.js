import React from 'react';
import Renderer from 'react-test-renderer';

import App from './app';

test('renders landing message correctly', () => {
  const app = Renderer.create(<App/>).toJSON();
  expect(app).toMatchSnapshot();
});