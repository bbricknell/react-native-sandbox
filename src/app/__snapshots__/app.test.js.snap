// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing message correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "marginBottom": 10,
      }
    }
  >
    Welcome to React Sandbox
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
      }
    }
  >
    Cmd+R to reload
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
      }
    }
  >
    Cmd+D for dev tools
  </Text>
</View>
`;
